src/services/alert/adapter/index.js

```diff
@@ -0,0 +1,17 @@
+class Adapter {
+	constructor() {
+		if (new.target === Adapter) {
+			throw new TypeError('Cannot construct Abstract instances directly');
+		}
+	}
+
+	/**
+	 * Return
+	 * @param {String} instances
+	 */
+	async getMessage(instances) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['instances' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4499680)